### ISSUE TITLE
Combined Custom Model and Image

### ISSUE THREAD

**Issue by DoctorWiz** [xLightsSequencer/xLights#3825]
For many of the _"Daycor"_ and _"Impression"_ (printed) coro props I like to also add an 'Image' model and line it up with the 'Custom' [pixel prop] model.  It would be great though if these 2 things could be automatically linked, lined up, positioned, and scaled together, and be just one model.  As an example, I combine an image of a starlite mint along with my mint spinner models from Boscoyo and Gilbert Engineering.
Of course if you are creating your own new custom model from scratch, you would typically do that by loading a picture of it into the model editor and clicking where all the pixel holes are.  **So why not just save that image along with the model data?**
And for vendor provided printed props, they could include an image with the xModel file download, and the image filename could be specified in the .xmodel file.  (And the size in inches/cm and other useful info could be included too!)
Currently I have to take a picture of the model or go swipe the image from their store site, cut out the background, make the watermark less noticeable, adjust lighting, contrast, and keystoning, and figure out the correct scaling and X Y offset of the image to the model.  And it never lines up quite right, so then I have to tweak the model data and adjust the pixel positions.  Then when repositioning it in the layout, I have to make sure to select and move both the model and its image at the same time.
And even if it is a plain white coro model, it would still be useful to have a white silhouette image of the shape with a transparent background.  Snowflakes in particular come to mind here, but others too.
And a related enhancement would be be able to specify the dimensions (inches/cm) and/or the X,Y scale in the model editor and save that in the .xmodel file as well.  So when dropping a new custom model into a layout, it could automatically size/scale it correctly.

**Comment by cybercop23** [comment]
IDK the feasability of this. Others have aked the same thing and even a way to "lock" 2 native (or custom) models like the EFL Fountain and base Spinner.

@keithsw1111 @computergeek1507 @AzGilrock 

maybe we can move this to a discussion and vote.

**Comment by computergeek1507** [comment]
It is a feature that would be nice to have but it's not an easy problem to solve. Most "easy" features get added pretty quickly

**Comment by AzGilrock** [comment]
I don't think it would be that hard as long as we don't try to lock 2 models together.  I think a better solution would be to have an image property that is part of the model.  Probably would need a whole "Image" section of parameters since it would need position offsets and brightness parameters.

**Comment by derwin12** [comment]
In the next update

**Comment by DoctorWiz** [comment]
Thanks for adding this!  You guys rock!
